common.mk

```diff
@@ -19,3 +19,7 @@ endif
 ifeq ($(findstring Python 3.6, $(shell python --version)),)
 $(error Please run make commands from a Python 3.6 virtualenv)
 endif
+
+ifneq ($(shell python -c 'import dyndbmutex' && echo yes), yes)
+$(error The dyndbmutex package is missing. Installing build requirements from requirements-dev.txt should fix that)
+endif
```

Comment:
please add newline

Reply:
Will do.

Reply:
Is this necessary?  Will people on the cmd line be running this?

Reply:
I'd say that if a recipe in the Makefile depends on it, the Makefile should check for it.

As a newcomer to the project I found these checks very helpful.

If the fix required installing something that would *only* be used by the `lock-stage` target, I'd say we should drop it. But since every developer has to install the build requirements, this is helpful.

Reply:
But I thought `dyndbmutex` is only used by lock-stage...

Reply:
Correct, but the fix (installing the dev reqs) is needed by everyone.

Reply:
Maybe I should list the actual `pip` command line here?

Reply:
I still don't get it.  We don't go and validate the presence of the vast majority of packages in requirements.txt in the Makefile.  There's nothing special about this requirement that makes it necessary to validate its presence.

Anyway, not a big deal, if you feel strongly about it, go ahead and do it.

Reply:
I'll remove it.